wagtailio/project_styleguide/templates/patterns/includes/svg_sprites.html

```diff
@@ -1,54 +1,199 @@
.017-1.722-.038-.023-.04-.02-.939 1.765zm-16.666 8.057l-1.978-.296-.006.038-.004.039 1.988.22zm-8.441 76.4l.494 1.938 1.342-.342.152-1.376-1.988-.22zM18.79 138.707l-.494-1.938-.038.01-.037.011.57 1.917zm-5.66 17.625l-1.617 1.176.026.036.028.035 1.564-1.247zm48.158 60.382l1.716 1.028.713-1.19-.865-1.085-1.564 1.247zm-39.345 65.639l-1.715-1.029-.027.044-.024.046 1.766.939zM30 299.019l-.296 1.978.038.005.039.005.22-1.988zm76.4 8.441l1.938-.494-.342-1.342-1.376-.152-.22 1.988zm18.992 74.481l-1.938.495.01.037.011.037 1.917-.569zm17.625 5.66l1.176 1.618.042-.031.041-.033-1.259-1.554zM151.17 381l1.259 1.555.726-.588.015-.935-2-.032zm34.16-78.198l1.362 1.464 3.246-3.019-4.412-.435-.196 1.99zm36.42-33.56l.937 1.767.902-.479.141-1.01-1.98-.278zm39.603-74.271l1.327 1.496.856-.759-.221-1.123-1.962.386zm254.45 90.752l-1.995.14.099 1.414 1.366.375.53-1.929zm-86.753-45.622l-1.764.942 1.062 1.99 1.848-1.293-1.146-1.639zm-179.304 59.92l.301 1.977 1.823-.278-.129-1.839-1.995.14zm-70.603 86.777l1.997-.102-.001-.019-.001-.019-1.995.14zm-79.056-173.659c-3.58-51.192 35.456-96.098 86.648-99.677l-.279-3.991c-53.396 3.734-94.093 50.551-90.36 103.947l3.991-.279zm86.648-99.677c43.609-3.05 82.738 25.407 95.698 65.866l3.809-1.22c-13.49-42.116-54.238-71.822-99.786-68.637l.279 3.991zm98.583 67c15.582-8.761 32.078-14.973 49.697-16.205l-.279-3.99c-18.388 1.286-35.463 7.76-51.379 16.708l1.961 3.487zm49.697-16.205c1.582-.11 3.408-.038 5.381.043 1.915.078 3.977.165 5.823.035l-.279-3.99c-1.582.111-3.408.039-5.381-.042-1.915-.078-3.977-.165-5.823-.036l.279 3.99zm12.775-.88l18.537-30.586-3.421-2.074-18.537 30.587 3.421 2.073zm18.506-30.538c2.732-4.227 2.695-8.867.754-12.605-1.918-3.697-5.655-6.443-10.194-7.124l-.594 3.956c3.296.494 5.921 2.472 7.238 5.01 1.295 2.496 1.369 5.602-.564 8.593l3.36 2.17zm-9.513-19.739l-75.543-8.501-.447 3.975 75.542 8.501.448-3.975zm-73.829-7.007l-18.991-74.482-3.876.988 18.991 74.482 3.876-.988zm-19.012-74.557c-2.668-8.99-13.104-12.241-20.718-6.708l2.351 3.236c5.503-4 12.692-1.59 14.532 4.61l3.835-1.138zm-20.8-6.646l-59.524 48.098 2.514 3.111 59.524-48.098-2.514-3.11zm-57.25 47.931l-66.496-39.285-2.034 3.444 66.496 39.285 2.034-3.444zm-66.574-39.329c-8.31-4.42-18.192.252-19.583 9.527l3.956.593c.959-6.397 7.743-9.782 13.749-6.588l1.878-3.532zm-19.593 9.604l-8.441 76.4 3.976.439 8.441-76.4-3.976-.44zm-6.947 74.681l-74.482 18.992.988 3.876 74.482-18.992-.988-3.876zM18.22 136.79c-8.99 2.668-12.241 13.104-6.708 20.718l3.236-2.351c-4-5.503-1.59-12.693 4.61-14.533l-1.138-3.834zm-6.654 20.789l48.158 60.382 3.127-2.494-48.158-60.382-3.127 2.494zm48.006 58.107l-39.345 65.638 3.43 2.057 39.346-65.639-3.431-2.056zm-39.396 65.728c-4.42 8.31.252 18.192 9.527 19.583l.593-3.956c-6.397-.959-9.782-7.743-6.588-13.749l-3.532-1.878zm9.604 19.593l76.4 8.441.439-3.976-76.4-8.441-.44 3.976zm74.681 6.947l18.992 74.482 3.876-.989-18.992-74.481-3.876.988zm19.013 74.556c2.668 8.991 13.104 12.242 20.718 6.709l-2.352-3.236c-5.502 3.999-12.692 1.59-14.532-4.611l-3.834 1.138zm20.801 6.6
```

Comment:
Good spot! I've removed the class references as the refactor moves away from using that method. Unfortunately the paper plane svg does not follow the format of the other SVGs so it will need more work if we want to be able to animate this one.